README.md

```diff
@@ -9,7 +9,15 @@ TBD
 
 ### Upgrade script
 
-TBD
+1. Deploy a new instance of GIVpower by script
+```
+forge create --rpc-url <RPC_URL> --private-key <PRIVATE_KEY> contracts/GIVpower.sol:GIVpower 
+```
+> **_NOTE:_**  The verification part (--verify --etherscan-api-key <ETHERSCAN_API_KEY>) is optional  
+2. Grab the address of new GIVpower instance deployed above and use it in calling `upgrade` method on `ProxyAdmin` contract by a privilaged account.
```

Comment:
I tried to do this using a script though I hardcode the addresses. What do you think of using the ffi cheatcode to feed the deployed new implementation address?

Other option is adding instructions on how to call the upgrade function using the cast command.

Reply:
Both may work. I just tailored this for Giveth use case where the Giveth admin is the multi-sig